### DISCUSSION TITLE
how to apply 'fixed:true' behaviour in the checkbox of 'show-select' column (using v3.5.15) ?

### DISCUSSION THREAD

**Discussion by IshmamAbir** [vuetifyjs/vuetify#19648] [Q&A] [answered]
### for example:  
My table tag will look like this.
```
<v-data-table-server
        :headers="headers"
        :items="serverItems"
        show-select
        item-value="name"
 />
```
And one of headers item contain an object
```
    {
          title: 'Dessert (100g serving)',
          key: 'name',
          fixed: true,
          width: '120',
     }
```

I would like to have the same behaviour of this column headers in the selection checkbox column.

### How can I achieve this?

**Comment by AntonioDell** [comment, marked as answer]
Hi,
it is possible to just define the data-table-select header together with your other headers and set it's fixed property to true:
`const headers = [{key: "data-table-select", fixed: true}, ...other headers]`

Try it in the playground:
https://play.vuetifyjs.com/#eNp1UcFugzAM/RWLSzepjG5HxCpN2qXn7db2kAa3RA0EJYa2q/rvcyAI0LRLlLwXP/s9b++RszL5qOuXtsEojTLCstaCcL2rALI2FnXdXbuHNBUJVaENUAfmgkRM4qBxAAHSAkWO1r3vonDbRRNWcRfPObQt2o1/TXlXmEvsUKOkEfQ1cSt0g1xYiRJniqU5KO2Zo9BuRjm6dUQprvFF5VSk8Lpa1dfxzzpLpi4GuwzO/XqkjyNLJjHx00mragKH1PRpcaEjCNbhHba95h3OeEthMTYLNhdLOKor5imQbRAey/B/GJEUaeTCT3ScGcETWzhxP9uq6vS8CN8hyPt0Rmww/bYK0L/q3+h4krkWTbC+cO8tDxYnG5zYHCT8IKyxkSgtinLU9qpMfJkSgetvoE11gm67QAZkgfIMhbHqx29AAwdstGavcMBCtMrYPzNlSb8Fzj96LKtG6+7Y/wIfNeSr

**Reply by IshmamAbir** [reply]
thanks for this minimal solution. worked fine.
